source/codegen/metadata/nidmm/attributes.py

```diff
@@ -768,6 +771,7 @@
         },
         'enum': 'AutoZero',
         'lv_property': 'Configuration:Measurement Options:Auto Zero',
+        'enum': 'AutoZero',
```

Comment:
`'enum': 'AutoZero'` is already specified two lines up.

Reply:
fixed